src/App.js

```diff
@@ -5,18 +5,19 @@ import firebase from 'firebase/app';
 import 'firebase/firestore';
 import 'firebase/auth';
 import 'firebase/analytics';
+import PropTypes from 'prop-types';
 
 import { useAuthState } from 'react-firebase-hooks/auth';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 firebase.initializeApp({
-  apiKey: "AIzaSyAyvwkmdni8c_dk2wkEqCAKQJVkD8ghStI",
-  authDomain: "project-2-5e162.firebaseapp.com",
-  projectId: "project-2-5e162",
-  storageBucket: "project-2-5e162.appspot.com",
-  messagingSenderId: "856397006394",
-  appId: "1:856397006394:web:dbd9a96b16f4e6f0c19703",
-  measurementId: "G-5M38L4N4RJ"
+  apiKey: "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
```

Comment:
Don't change my Firebase credentials.

Reply:
you can rebase this with firebase-config.json and it should be good to go.